### PR TITLE
updated contributing

### PR DESCRIPTION
**Pull request by isabelshic**
added hahahaha

### CODE REVIEW

**Comment by asewnath** [comment]
:D

**Comment by asewnath** [comment]
Very welcoming!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Contributing :)
 
+hahahaha
+
```
Comment (ashiklom) [inline CONTRIBUTING.md:4]: Please use sentence case when writing prose, and don't forget punctuation.

**Review by ashiklom — CHANGES_REQUESTED** [review]
Changes requested.

File: CONTRIBUTING.md (hunk 1 above)
Comment (isabelshic) [reply CONTRIBUTING.md:4]: no

**Pull request merged**